### PR TITLE
dump the crash log information on to the console

### PR DESCRIPTION
**Pull request by mkasturi18**
Signed-off-by: MadhaviKasturi <mkasturi@redhat.com>

dump the crash log on to the console if any

PFA, the [logs](http://magna002.ceph.redhat.com/ceph-qe-logs/madhavi/PRs/dump_log/)

### CODE REVIEW

File: rgw/v2/utils/utils.py [hunk 1]
```diff
@@ -42,6 +42,17 @@ def exec_shell_cmd(cmd, debug_info=False):
     except Exception as e:
         log.error("cmd execution failed")
         log.error(e)
+        # dump the crash log information on to the console, if any
+        ceph_version_id, ceph_version_name = get_ceph_version()
```
Comment (udaysk23) [inline rgw/v2/utils/utils.py:46]: Can you use `_` instead of `ceph_version_id` since we are not using that variable further in this method

Comment (mkasturi18) [reply rgw/v2/utils/utils.py]: @udaysk23 , sure made the changes

**Review by psathyan — COMMENTED** [review]
The logic looks good... 

Would it make sense to have a separate method to print the crash contents?

**Comment by mkasturi18** [comment]
@psathyan , sure added a separate method

**Review by psathyan — APPROVED** [review]

**Pull request merged**